1-JavaScript/Week3/homework/ex4-shoppingCart.js

```diff
@@ -21,8 +21,16 @@ you have more than 3 items in your shopping cart the first item gets taken out.
 -----------------------------------------------------------------------------*/
 const shoppingCart = ['bananas', 'milk'];
 
-function addToShoppingCart(/* parameters go here */) {
+function addToShoppingCart(groceryItem) {
   // TODO complete this function
+  shoppingCart.push(groceryItem);
+  if (shoppingCart.length > 3) {
+    shoppingCart.shift();
+  }
+  for (let item in shoppingCart) {
+    shoppingCart[item];
+  }
```

Comment:
only two small issues. The first one is this block is not necessary. You already did the necessary operations to solve the problem. The second issue is, like in exercises 1 and 2, you need to return a string and this method is not returning anything.